### DISCUSSION TITLE
7.8.1 (Grid effect enchantments and bugfixes)

### DISCUSSION THREAD

**Discussion by T-Troll** [T-Troll/alienfx-tools#257] [Announcements]
### **Warning!**
**Configuration format is changed! Backup your configuration using `alienfx-config`!**
- Starting from 7.8.0, Grid effects configuration settings changed. You will lose all defined colors, but two first and colors flags in case you roll back.
- You will lose all fan settings and fan sensor names in case you roll back from this version to 7.5.0 or early!
- You will lose all sensor names and sensor settings for `AlienFX-Mon` tool if you upgrade from 7.5.0 or early!

###  Changes into this release:

`AlienFX-GUI`:
- New grid effect type - "Fill".
- A lot of small changes into Grid effect logic:
  - Effects now cycle through all colors, including first one, then last-to first before stop
  - Some zone borders altered
  - Zone size correction for reverse gauge type
- Multi-cell lights not processed as many lights with different position, not the one at upper left.
- Bug fixed: Exception if gauge zone start effect first time for some cases.
- Bug fixed: Incorrect zone processed into Grid Effect (stuck to first active).
- Bug fixed: Screen grid color prediction not updated correctly in some cases.

Other:
- Beginner's guide updated.

<hr /><em>This discussion was created from the release <a href='https://github.com/T-Troll/alienfx-tools/releases/tag/7.8.1'>7.8.1 (Grid effect enchantments and bugfixes)</a>.</em>

**Comment by guidbp** [comment]
The .msi file is installing the 7.8.0 version.

**Reply by guidbp** [reply]
.zip is OK

**Reply by T-Troll** [reply]
This should not be like this, so i update .msi to be sure. Now it has 7.8.1 as well.

**Reply by guidbp** [reply]
Worked ... tks!!
